classes/01class/exercises/c01-aws01/frdvo/README.md

```diff
@@ -0,0 +1,318 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+
+### Check the current region
+$ aws configure get region
+ap-southeast-2
+**Generate SSH Key for JumpBox**
+`aws ec2 create-key-pair --key-name dajb --output text > dajb.pem`
+**#protect the key, read only, only to my user**
+chmod 400 da.pem
+**Create Security Group**
+`aws ec2 create-security-group --group-name SSHOpen --description "SG with port 22 open to the world"`
+{
+    "GroupId": "sg-063fc9a232e4e2a60"
+}
+
+
+
+**Add rule to allow Port TCP 22 inbond connections**
+`aws ec2 authorize-security-group-ingress \`
+    `--group-name SSHOpen \`
+    `--protocol tcp \`
+    `--port 22 \`
+    `--cidr 0.0.0.0/0`
```

Comment:
to keep things more secure, avoid having a rule opening a port to the world(unless your SG is being used by a public service). Can you think in a way to have this SG more secure?

Reply:
Yes, you are right. I should restrict to my IP address only. If my IP change, I should update it.